README.md

```diff
@@ -1,5 +1,7 @@
 # Welcome to the Mixed Reality Toolkit
 
+![](Images/Branding/XRTK_Logo_948x474.png)
```

Comment:
This is kinda a random image size. Could we update this to be something standard?